packages/headless/src/controllers/folded-result-list/headless-folded-result-list.ts

```diff
@@ -113,6 +118,15 @@ export function buildFoldedResultList(
   return {
     ...controller,
 
+    loadAll: (collection) =>
+      dispatch(
+        loadAll(
```

Comment:
I would prefer if the action name was more explicit, because `All` does not provide any context. I would suggest: `loadCollection`